app/views/organizations/index.html.erb

```diff
@@ -1,29 +1,31 @@
-<h1>Organizations</h1>
+<div class="mb4 ph2">
+  <div class="mw7 center">
+    <h1 class="f3 fw7 mt0 mb3">Organizations</h1>
 
-<table>
-  <thead>
-    <tr>
-      <th>Name</th>
-      <th>Contact</th>
-      <th>Email</th>
-      <th>Phone</th>
-      <th colspan="3"></th>
-    </tr>
-  </thead>
-
-  <tbody>
     <% @organizations.each do |organization| %>
-      <tr>
-        <td><%= organization.name %></td>
-        <td><%= organization.contact %></td>
-        <td><%= organization.email %></td>
-        <td><%= organization.phone %></td>
-        <td><%= link_to 'Show', organization %></td>
-        <td><%= link_to 'Edit', edit_organization_path(organization) %></td>
-        <td><%= link_to 'Destroy', organization, method: :delete, data: { confirm: 'Are you sure?' } %></td>
-      </tr>
+      <div class="bg-white shadow pa3 pa4-l">
+        <h2 class="f4 fw7 mb3"><%= organization.name %></h2>
+        <div class="cf">
+          <div class="fl-l w-50-l pr4-l">
+            <h3 class="fw7 mb1">Address</h3>
+            <div class="mb3">123 Placeholder Ave<br>Portland, OR 97202</div>
```

Comment:
This is excellent 💫 I created issue #103 to get this information added to the db.

Reply:
Great, thanks!